packages/dx-react-grid/src/plugins/table-fixed-columns.jsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {
+  Getter,
+  Template,
+  Plugin,
+  TemplatePlaceholder,
+  TemplateConnector,
+} from '@devexpress/dx-react-core';
+import {
+  FIXED_COLUMN_BEFORE_SIDE,
+  FIXED_COLUMN_AFTER_SIDE,
+  isFixedCell,
+  getFixedSide,
+  getFixedColumnKeys,
+} from '@devexpress/dx-grid-core';
+
+const CellPlaceholder = props => <TemplatePlaceholder params={props} />;
+
+const pluginDependencies = [
+  { name: 'Table' },
+];
+
+export class TableFixedColumns extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = { sizes: {} };
+  }
+
+  render() {
+    const {
+      beforeColumnNames,
+      afterColumnNames,
+      beforeColumnTypes,
+      afterColumnTypes,
+      cellComponent: Cell,
+    } = this.props;
+
+    const fixedColumnNames = [...beforeColumnNames, ...afterColumnNames];
+    const fixedColumnTypes = [...beforeColumnTypes, ...afterColumnTypes];
+
+    const fixedColumnKeysComputed = ({ tableColumns }) => [
+      ...getFixedColumnKeys(tableColumns, beforeColumnNames, beforeColumnTypes),
+      ...getFixedColumnKeys(tableColumns, afterColumnNames, afterColumnTypes),
+    ];
+
+    return (
+      <Plugin
+        name="TableFixedColumns"
+        dependencies={pluginDependencies}
+      >
+        <Getter name="fixedColumnKeys" computed={fixedColumnKeysComputed} />
```

Comment:
I think maybe we should embed this information into the tableColumns getter? E.g. by providing `fixed: true` field.